pkg/controllers/managementuser/rbac/handler_base.go

```diff
@@ -116,11 +118,11 @@ func Register(ctx context.Context, workload *config.UserContext) {
 		ResourceQuotaLister: workload.Core.ResourceQuotas("").Controller().Lister(),
 		LimitRange:          workload.Core.LimitRanges(""),
 		LimitRangeLister:    workload.Core.LimitRanges("").Controller().Lister(),
-		ProjectLister:       workload.Management.Management.Projects(workload.ClusterName).Controller().Lister(),
+		ProjectLister:       management.Management.Projects(workload.ClusterName).Controller().Lister(),
 	}
 
 	workload.Core.Namespaces("").AddLifecycle(ctx, "namespace-auth", newNamespaceLifecycle(r, sync))
-	workload.Management.Management.RoleTemplates("").AddHandler(ctx, "cluster-roletemplate-sync", newRTLifecycle(r))
+	management.Management.RoleTemplates("").AddHandler(ctx, "cluster-roletemplate-sync", newRTLifecycle(r))
 }
 
 type manager struct {
```

Comment:
Are we only doing 2.6 and not 2.7 for the test run ?

Reply:
I'm planning on doing both and am partway through forward porting to 2.7. Just waiting on this.